test/spec/controllers/eventDetails.js

```diff
@@ -1,65 +0,0 @@
-'use strict';
```

Comment:
It's fine if you keep the file deleted locally, just don't commit that deletion. Please revert this change. If you need help with the git commands, let me know.